controllers/genreController.js

```diff
@@ -0,0 +1,25 @@
+const { Genre } = require('../models');
+
+/**
+ * Serves an array of movie genres
+ * @return  genres
+ */
+module.exports.readAll = async (req, res) => {
+  const allGenres = await Genre.find({});
+
+  if (res) res.json(allGenres);
+  else return allGenres;
+};
+
+module.exports.readOne = async (req, res) => {
+  const foundGenre = await Genre.findById(req.params.id);
+  res.json(foundGenre);
```

Comment:
Should this return the res.json explicitly and check to see if it actually found one, like the function above?
```suggestion
  return res.json(foundGenre);
```

Reply:
Nope.